google/devtools/clouddebugger/v2/clouddebugger_gapic.yaml

```diff
@@ -0,0 +1,167 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.devtools.clouddebugger.spi.v2
+  python:
+    package_name: google.cloud.devtools.clouddebugger.v2
```

Comment:
Recent rename: `google.cloud.gapic.devtools.clouddebugger.v2 `

@geigerj just fixed the GAPIC config generator to address this in the future.